### DISCUSSION TITLE
UTM Beta 4.5.1 error, VMs don't run

### DISCUSSION THREAD

**Discussion by macuserguru** [utmapp/UTM#6280] [UTM Help (iOS)]
I have installed on M1 MacMini (Sonoma 14.4.1) UTM 4.4.4 (from App Store) and UTM Beta 4.5.1 too.
Now if I want to start my VMs with 4.5.1 an error message comes - with 4.4.4 they run without problem. With 4.5.0 they had run without problem too.
All OS in VM are native ARM aarch64 versions.

What are wrong now?

“Error
The process could not be completed. The process is not authorised“

![UTM Error Beta 4 5 1](https://github.com/utmapp/UTM/assets/4521744/dacb03f4-ad36-4748-bc1a-af1ddcc8d111)

**Comment by Odaeus** [comment]
In case you haven't checked already, does UTM have full access e.g. under Datenschutz & Sicherheit -> Festplattenvollzugriff?

**Comment by macuserguru** [comment]
It was configured correctly.
The error is still present.
I have now extracted the system.log at the time in question and searched for "UTM"
The only error that appears several times:
"standard 14:25:13.994559+0200 QEMUHelper Error starting QEMU: Error Domain=NSPOSIXErrorDomain Code=1 "Operation not permitted - error 14:25:13.994522+0200 kernel exec of /Applications/UTM (v4.5.1 Beta). app/Contents/XPCServices/QEMUHelper.xpc/Contents/MacOS/QEMULauncher.app/Contents/MacOS/QEMULauncher denied since it was quarantined by Safari;E1.....C5 and not approved by Gatekeeper, qtn-flags was 0x00000183"

One of the components in the app container appears to be blocked by the gatekeeper.
It is possible that I downloaded this package with Safari and not with Firefox as usual.

**Comment by macuserguru** [comment]
Addendum: Downloaded the same file with Firefox and UTM (v4.5.1 Beta).app now works, VMs can be started.
